sdpdata.json

```diff
@@ -57,24 +57,24 @@
+      "lineText": "m=audio 9 RTP\/SAVPF 111 103 104 9 0 8 106 105 13 126",
+      "description": "<p><em>m<\/em> means it is a media line –  it condenses a lot of information on the media attributes of the stream. In this order, it tells us:<ul><li><em>audio<\/em>- the media type is going to be used for the session (media types are registered at the IANA),<\/li><li><em>9<\/em> – the port is going to be used for SRTP (and for RTCP if RTCP multiplex is supported by the other peer) should always be 9 in webrtc to indicate the \"discard\" protocol.,<\/li><li><em>RTP\/SAVPF<\/em> - the transport protocol is going to be used for the session, and last but not least<\/li><li><em>111 103 104 0 8 106 105 13 126<\/em> - the media format descriptions are supported by the browser to send and receive media.<\/li><\/ul>RTP\/SAVPF is defined in <a href=\"http:\/\/tools.ietf.org\/html\/rfc5124\">RFC5124<\/a>. In short it requires the use of SRTP and SRTCP and RTCP Feedback packets.<\/p><p>The media format descriptions, with protocol RTP\/SAVPF, gives the RTP payload numbers which are going to be used for the different formats.&nbsp; Payload numbers lower than 96 are mapped to encoding formats by the <a href=\"http:\/\/www.iana.org\/assignments\/rtp-parameters\/rtp-parameters.xhtml\">IANA<\/a>.&nbsp;In our SDP<em>0<\/em>maps to G711U and<em>8<\/em>to G711A. Format numbers larger than 95 are dynamic and there are <em>a=rtpmap:<\/em>&nbsp;attribute to map from the RTP payload type numbers to media encoding names.&nbsp; There are also<em>a=fmtp:<\/em> attributes which specify format parameters<\/p>"
     },
     {
       "display": "line",
       "lineNum": "7",
       "indent": "1",
       "section": "Audio Lines",
-      "lineText": "c=IN IP4 217.130.243.155",
-      "description": "<em>c<\/em> is a connection line. This line gives the IP from where you expect to send and receive the real time traffic. As ICE is mandatory in WebRTC the IP in the c-line is not going to be used."
+      "lineText": "c=IN IP4 0.0.0.0",
+      "description": "<em>c<\/em> is a connection line. This line gives the IP from where you expect to send and receive the real time traffic. As ICE is mandatory in WebRTC the IP in the c-line is not going to be used. Firefox and Chrome both send 0.0.0.0"
     },
     {
       "display": "line",
       "lineNum": "8",
       "indent": "1",
       "section": "Audio Lines",
-      "lineText": "a=rtcp:51472 IN IP4 217.130.243.155",
-      "description": "This line explicitly specifies the IP and port that will used for RTCP, not derived from the base media port. Note that is the same port as for SRTP as RTCP multiplex is supported."
+      "lineText": "a=rtcp:9 IN IP4 0.0.0.0",
```

Comment:
yes, while there are not ICE candidates some valid values must be given to the port and the IP in the "rtcp" line, so '9' and '0.0.0.0' are the 'default' values. It would be also a good thing to add this to the explanation.